coral/src/app/pages/not-found/index.tsx

```diff
@@ -1,23 +1,55 @@
-import { Box, Typography } from "@aivenio/aquarium";
+import { Box } from "@aivenio/aquarium";
 import AuthenticationRequiredBoundary from "src/app/components/AuthenticationRequiredBoundary";
 import Layout from "src/app/layout/Layout";
+import { FileInput } from "src/app/components/FileInput";
+import { ChangeEvent, useState } from "react";
 
 const NotFound = () => {
+  const [file, setFile] = useState<File>();
+
   return (
     <AuthenticationRequiredBoundary>
       <Layout>
         <Box role="main" display={"flex"} flexDirection={"column"} gap={"5"}>
-          <Typography.Heading color={"secondary-100"}>
-            Page not found
-          </Typography.Heading>
+          <FileInput
+            valid={true}
+            labelText={"Please upload file"}
+            helperText={"There was an error"}
+            noFileText={"No" + " file chosen"}
+            fileName={file?.name}
+            onChange={(event: ChangeEvent<HTMLInputElement>) =>
+              setFile(event?.target?.files?.[0])
```

Comment:
🤔  Not really sure what you mean by this (like, my 🧠  doesn't compute, not like that you don't make sense ^^).

Reply:
Sorry if I've been a little too abstract ^^' This is what I mean: 

https://user-images.githubusercontent.com/20607294/212068789-2be7fa80-537a-4cd0-a7e7-1c0d6d94925e.mov

You can see that maintaining the `fileName` state is not necessary, as the `input` itself maintain it, and we can get it through the `files` attribute. If the `multiple` attribute is not present, then I think it will always be an array of 1 file in that `files` input state. If we need multiple file uploads, we would need to map over `files` ^^

Reply:
OMG thank you, that was super helpful!!!

YES that is a great improvement!! 

I added the ref in the end, to make mouse clicks work correctly. Didn't think about using it some where else. 

NICE!